app/api/orders.py

```diff
@@ -287,10 +287,12 @@ def before_update_object(self, order, data, view_kwargs):
                                                      "You cannot update the status of a cancelled order")
 
         elif current_user.id == order.user_id:
-            if order.status != 'pending':
+            print(order.status)
+            if order.status != 'initializing':
```

Comment:
This is again wrong. Pending orders can be updated. Users can pay and complete them.